supply-chain/config.toml

```diff
@@ -0,0 +1,991 @@
+
+# cargo-vet config file
+
+[cargo-vet]
+version = "0.10"
```

Comment:
how does the config work here ?

Reply:
It's auto generated on `cargo vet init` and refreshed after a run of `cargo vet`